documentation/modules/ROOT/pages/2deploy-microservices.adoc

```diff
@@ -55,7 +55,6 @@ NOTE: Your very first Docker build will take a bit of time as it downloads all t
 [source,bash]
 ----
 cd customer/java/springboot
-mvn clean package
```

Comment:
To not confuse the reader (Java reader) I would add a NOTE before this snippet explaining that we are using multi-stage builds.